app/concepts/matestack/ui/core/ins/ins.haml

```diff
@@ -0,0 +1,5 @@
+%ins{@tag_attributes.merge({:cite => options[:cite], :datetime => options[:datetime]})}
```

Comment:
Hey @lumisce, could you move the `@tag_attributes.[...]` to the `ins.rb`? It works well the way it its, but we're trying to keep the `.haml` file as minimal as possible :) feel free to look at other components and ask questions on our [chat](https://gitter.im/basemate/community) if you get stuck!